django_elasticsearch_dsl/search.py

```diff
@@ -27,13 +32,26 @@ def to_queryset(self, keep_order=True):
             s = s.execute()
 
         pks = [result.meta.id for result in s]
+        queryset = queryset.filter(pk__in=pks)
 
-        qs = self._model.objects.filter(pk__in=pks)
-
-        if keep_order:
+        if keep_search_order:
             preserved_order = Case(
                 *[When(pk=pk, then=pos) for pos, pk in enumerate(pks)]
             )
-            qs = qs.order_by(preserved_order)
+            queryset = queryset.order_by(preserved_order)
+
+        return queryset
+
+    def _get_queryset(self):
```

Comment:
You can make this api public!

Reply:
I made it private so it would be less confusing with IDE hinting for example, as we have public methods that are mostly related to filtering and search object manipulation. What would be the public usage of `get_queryset`?